DanielSantos01/money-map-backend
# feature/jestTest: test implemented to user CRUD

###

**What I Did:**
- implemented Jest tests to create user cases

**How to Test:**
- update your .env file (see in Notion)
- run `npx jest`
